src/views/Navbar/NavbarDropdown/NavbarDropdown.js

```diff
@@ -57,6 +57,50 @@ export default function NavbarDropdown(props) {
 
   // Looks into the 'sections' property exported by each page component to see
   // what anchor-links that page makes available
+  //Polyfill for Array.prototype.find (method is not compatible with Internet Explorer)
```

Comment:
Nice, great work finding the IE bug!

Only comment here is, polyfills should be not be placed within a React component. A good idea would be to move it to `src/utilities/polyfill.js`, then just throw in a 
```js
import './utilities/polyfill'
```
at the top of `src/index.js`